passport/facebook.js

```diff
@@ -0,0 +1,44 @@
+const credentials = require('../credentials.json');
```

Comment:
Not sure if calling dir `passport` makes sense since it is just name of the tool you're using, not any kind of functionality. I believe `auth` would be more appropriate.